tsnet/tsnet.go

```diff
@@ -144,8 +149,6 @@ func (s *Server) Close() error {
 	s.listeners = nil
 
 	// Perform a best-effort final flush.
-	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
-	defer cancel()
 	s.logtail.Shutdown(ctx)
```

Comment:
The time might've been exhausted at this point. Do the two five second things concurrently?

Reply:
Done